_doc-build-linux/action.yml

```diff
@@ -445,6 +465,13 @@ runs:
       shell: bash
       run: |
         mv documentation-html.zip ${{ env.EXPECTED_DOWNLOAD_DIR }}
+
+        if [[ ${{ inputs.skip-pdf-build }} == 'true' ]];
+        then
+          echo "Not attaching PDF since its generation has not being requested."
+          exit 0;
+        fi
+
         if [ -e  "${{ env.EXPECTED_BUILD_DIR }}/latex/${{ env.PDF_FILENAME }}" ]; then
             cp ${{ env.EXPECTED_BUILD_DIR }}/latex/${{ env.PDF_FILENAME }} ${{ env.EXPECTED_DOWNLOAD_DIR }}
```

Comment:
```suggestion

        if [[ ${{ inputs.skip-pdf-build }} == 'true' ]]; then
          echo "Not attaching PDF since its generation has not being requested."
        elif [ -e  "${{ env.EXPECTED_BUILD_DIR }}/latex/${{ env.PDF_FILENAME }}" ]; then
            cp ${{ env.EXPECTED_BUILD_DIR }}/latex/${{ env.PDF_FILENAME }} ${{ env.EXPECTED_DOWNLOAD_DIR }}
```

Reply:
I have asumed here than `exit 0` will just exit this step (code snippet). Am I right?

Reply:
Since it is now a multiple-if statment, you do not need to worry